cmd/skuba/main.go

```diff
@@ -74,15 +73,3 @@ func printVersionStatement() {
 		fmt.Printf("** This is a tagged version (%s), but is not built in release mode (mode: %q.) **\n", skubapkg.Tag, skubapkg.BuildType)
 	}
 }
-
-// register adds a flag to local that targets the Value associated with the Flag named globalName in flag.CommandLine.
```

Comment:
Nit: the comment explaining what is happening with 'local' is not reflected in the new verbose.go file.